package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Template for generating Go module based on async api",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"no test specified yet\"",
+    "test": "npm run testComponents",
     "release": "semantic-release",
     "lint": "eslint --max-warnings 0 --fix --config .eslintrc .",
     "generate:readme:toc": "markdown-toc -i README.md",
     "generate:assets": "npm run generate:readme:toc",
-    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "testComponents": "rimraf test/temp && jest --coverage ./test/components"
```

Comment:
You do not put anything in `test/temp` so no need to remove it and using `rimraf`. I guess you saw it in other templates but this is related to not unit testing but a kind of integration testing where you actually run generator and then check if generated app matches the snapshot. Definitely another set of tests I recommend to add after this PR (example https://github.com/asyncapi/nodejs-template/pull/103)